compiler/ballerina-lang/src/main/java/io/ballerina/projects/Bootstrap.java

```diff
@@ -172,6 +173,7 @@ public void loadLangLibSymbols(CompilerContext compilerContext) {
         symbolTable.langXmlModuleSymbol = loadLangLibFromBala(XML, compilerContext);
         symbolTable.langBooleanModuleSymbol = loadLangLibFromBala(BOOLEAN, compilerContext);
         symbolTable.langQueryModuleSymbol = loadLangLibFromBala(QUERY, compilerContext);
+        symbolTable.langTransactionModuleSymbol = loadLangLibFromBala(RUNTIME, compilerContext);
```

Comment:
Why do we set the `langTransactionModuleSymbol` twice here? 
Shouldn't it be `langRuntimeModuleSymbol`?